config/locales/en/document_types.yml

```diff
@@ -0,0 +1,112 @@
+en:
+  default:
```

Comment:
We should switch these, so the file name corresponds to the namespace within. This should be safe, since we're not likely to have a `default` document type 🤞

Reply:
Note to self after call with Ben: it doesn't matter that the 'managed elsewhere' document types (`speech`, etc) would inherit the default properties. In fact, they should be left in `config/document_types.yml` rather than moved into I18n as they will be deleted later.

Reply:
Actually, not moving the the "managed elsewhere" document types is more work than expected, as we then have two ways of retrieving `label`/`description` 🤔 so code will end up something like this:

```
def description
  return t!("document_types.#{id}.description") if I18n.exists?("document_types.#{id}.description")

  hash["description"]
end
```

Is there any harm in transferring these psuedo-document-types across? I think it will make them easier to delete, as opposed to more difficult.

Reply:
Good point! 

On your comment: if it's easier to extract the label and description now, then let's go for it, as I agree doing some kind of switch is unnecessarily complicated.

@kevindew also pointed out that we should think carefully about whether any of these fields are 'inherent properties' of the object (like maybe label), rather than auxiliary things (definitely guidance). On that note: if extracting a field like `label` has difficult consequences (maybe in the tests), then perhaps we should continue to treat it as a property.